File: fibonacci.c
#include <stdio.h>
int fib(int a){
    if(a==0)
    return 0;
    if(a==1||a==2)
    return 1;
    else
    return (fib(a-1)+fib(a-2));
}

int main(){
    int n;
    printf("Enter a postion of a number in the fibonacci sequence: ");
    scanf("%d",&n);

    printf("The number in position %d is %d",n,fib(n));
    return 0;
}

File: fibonacci.py
def fib(n):
    if n==0 or n==1:
        return n
    else: 
        return fib(n-1)+ fib(n-2) #since we are getting the next fibonacci number, we add the previous two numbers.
    
print(fib(5))

'''fib(5)=fib(4)+fib(3)
   fib(4)=fib(3)+fib(2)
   fib(3)=fib(2)+fib(1)
   fib(2)=fib(1)+fib(0)'''
    
